experimental/packages/otlp-exporter-base/src/configuration/otlp-http-env-configuration.ts

```diff
@@ -0,0 +1,118 @@
+/*
+ * Copyright The OpenTelemetry Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+import { baggageUtils } from '@opentelemetry/core';
+import { diag } from '@opentelemetry/api';
+import { getSharedConfigurationFromEnvironment } from './shared-env-configuration';
+import { OtlpHttpConfiguration } from './otlp-http-configuration';
+
+function getHeadersFromEnv(signalIdentifier: string) {
```

Comment:
Hmm, I don't think they will grow much. I mostly just chose string as it was the easiest choice to work with and it does not require us to export a new type that could be difficult to extend later.

Do you think we should have a `SignalIdentifier` type? :thinking:

Reply:
one might think is a commodity but with the type the editor helps you to avoid passing a string which does not refer to any Signal. Is not wrong to not have a type here. I'll leave it to you

Reply:
I think I will leave it as-is - maybe I'm traumatized by making unintentional breaking changes in TypeScript libraries, but I feel like whatever type I can come up with won't be extendable in a safe way later on. :confused: 

The fact that libraries have to be type-compatible with other versions of themselves is kind of a mind-bender that did not exist in other languages I came from originally. This API will mostly be internal, so I'll opt to leave it like this for now. We can re-visit this choice in a later iteration -  I'm planning to have a final round of API review when the whole exporter restructuring is over to work out smaller things like this. :slightly_smiling_face: